tests/filterStale.test.js

```diff
@@ -0,0 +1,178 @@
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = [
+      { lastModified: dateC, status: "open", problems: ["It has been over 90 days since project's last modification."] }
+    ]
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns no identified stale open projects based on lastModified field', () => {
+    const projects = [
+      {lastModified: dateD, status: "open"},
+      {lastModified: dateE, status: "open"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = []
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns identified stale open projects based on endDate field', () => {
+    const projects = [
+      {endDate: dateA, status: "open"},
+      {endDate: dateB, status: "open"},
+      {endDate: todayDate, status: "open"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = [
+      { endDate: dateA, status: "open", problems: ["The project end date has passed, but project is marked open."] }
+    ]
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns no identified stale open projects based on endDate field', () => {
+    const projects = [
+      {endDate: dateB, status: "open"},
+      {endDate: todayDate, status: "open"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = []
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns identified stale ongoing projects based on lastModified field', () => {
+    const projects = [
+      {lastModified: dateC, status: "ongoing"},
+      {lastModified: dateD, status: "ongoing"},
+      {lastModified: dateE, status: "ongoing"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = [
+      { lastModified: dateC, status: "ongoing", problems: ["It has been over 90 days since project's last modification."] }
+    ]
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns no identified stale ongoing projects based on lastModified field', () => {
+    const projects = [
+      {lastModified: dateD, status: "ongoing"},
+      {lastModified: dateE, status: "ongoing"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = []
+
+    expect(filteredProjects).toEqual(expectedProjects)
+  })
+
+  it('returns identified stale ongoing projects based on endDate field', () => {
+    const projects = [
+      {endDate: dateA, status: "ongoing"},
+      {endDate: dateB, status: "ongoing"},
+      {endDate: todayDate, status: "ongoing"}
+    ]
+
+    const filteredProjects = filterStale(projects, todayDate)
+    const expectedProjects = [
+      { endDate: dateA, status: "ongoing", problems: ["The project end date has passed, but project is marked open."] }
```

Comment:
Looking at this, we might get into the situation where you want to modify the `problems` strings, and you'll end up needing to modify them in a bunch of different places (which is a pain and liable for mistakes). 
You could export the strings from the `lib.js` like 
```js
export const projectEndDatePastButNotClosed = "The project end date has passed, but project is marked open."
```

... and then use them here like: 
```js
{ endDate: dateA, status: "ongoing", problems: [projectEndDatePastButNotClosed] }
```